src/promptflow-evals/README.md

```diff
@@ -0,0 +1,23 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-tools)](https://pypi.org/project/promptflow-evals/)

```

Comment:
This link is incorrect pointing to tools

Reply:
Fixed (however, the evals are not on pypi yet)